src/main/scala/org/hammerlab/guacamole/reads/MappedRead.scala

```diff
@@ -42,7 +42,7 @@ case class MappedRead(
     alignmentQuality: Int,
     start: Long,
     cigar: Cigar,
-    mdTagString: String,
+    mdTagString: Option[String],
```

Comment:
Orthogonal change I was gong to suggest on the other PR: I'd rather `MappedRead` take an `Option[MdTag]`, and have an alternate ctor that allows passing an `Option[String]` and making an `Option[MdTag]` out of it